components/phone_com/app/phone_com.app.ts

```diff
@@ -1,13 +1,71 @@
 import { defineApp } from "@pipedream/types";
+import axios from "axios";
```

Comment:
It's best to use Pipedream's axios because it will return better debugging information.
```suggestion
import { axios } from "@pipedream/platform";
```